spec/vacancy_sources/vacancy_source/source/broadbean_spec.rb

```diff
@@ -4,7 +4,7 @@
   let(:response_body) { file_fixture("vacancy_sources/broadbean.xml").read }
   let(:response) { double("BroadbeanHttpResponse", success?: true, body: response_body) }
 
-  let!(:school1) { create(:school, name: "Test School", urn: "111111", phase: :primary) }
+  let!(:school1) { create(:school, name: "Test School", urn: "111111", phase: :primary, created_at: 1.week.ago) }
```

Comment:
Fixes an unrelated flaky test (ensuring we get the correct order on school creation/association).